src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -183,7 +162,7 @@ const NavHeader = (props) => {
                       ? {uri: topicDetail?.icon_path}
                       : TopicDefaultIcon
                   }
-                  style={styles.image}
+                  style={styles.image(followType)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style of the image is now dependent on the `followType`. This change seems fine as long as the `followType` prop is always provided and has a valid value.
```diff
- style={styles.image}
+ style={styles.image(followType)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->